armi/utils/__init__.py

```diff
@@ -806,17 +806,39 @@ def merge(self, *otherDictionaries) -> None:
 
 def safeCopy(src: str, dst: str) -> None:
     """This copy overwrites ``shutil.copy`` and checks that copy operation is truly completed before continuing."""
+    # Convert files to OS-independence
+    src = os.path.abspath(src)
+    dst = os.path.abspath(dst)
     waitTime = 0.01  # 10 ms
+    maxWaitTime = 1800  # 30 min
     if os.path.isdir(dst):
         dst = os.path.join(dst, os.path.basename(src))
     srcSize = os.path.getsize(src)
-    shutil.copyfile(src, dst)
-    shutil.copymode(src, dst)
+    # Both copy and cp -a accomplish the same thing (copy a file and its metadata)
+    if "win" in sys.platform:
+        cmd = f'copy "{src}" "{dst}"'
+    elif "linux" in sys.platform:
+        cmd = f'cp -a "{src}" "{dst}"'
+    else:
+        runLog.warning(
+            "Cannot perform ``safeCopy`` on files because ARMI only supports "
+            + "Linux and Windows."
+        )
+    os.system(cmd)
```

Comment:
The "else" case here will raise an error because the `cmd` variable doesn't exist yet.

Reply:
Oh, oops. 

This begs the question:
Do we want a ton of "if windows elif linux else kill the run" all throughout the code base? Why not just kill the run from the getgo?

Reply:
Well, we need something here, otherwise this is just invalid Python.  Hmmm... I will make a recommend.